### ISSUE TITLE
Initial Specifications

### ISSUE THREAD

**Issue by ttsukagoshi** [ttsukagoshi/axe-scan#2]
```
axe-scan test [options]
```

By default, search for `urls.txt` in the user's current directory and conducts accessibility tests against the listed URLs. STDOUT the list of page-by-page, element-by-element test results, where the result type is `violation` or `incomplete`:

```
axe-scan test > axe-results.csv
```

- Returns an error if urls.txt is not found. Optionally designate a path for the list of URLs.
- Designate a list of whitelisted alerts using the `--whitelist` option. The matching alerts will be excluded from the output

```
axe-scan test [--file | -F] path/to/urls-list.txt [--whitelist | -W] ./whitelisted-alerts.csv
```
User options will be determined in the following order: (1) `./axe-scan.config.json` file in the working directory, (2) `~/axe-scan.config.json` file in the user's home directory, and (3) default settings.

```
axe-scan config <key>=<value>
```

will save the setting to the local `./axe-scan.config.json` file. You can also manually save the JSON file to the directory of your choice.

```jsonc
{
  "axeCoreTags": ["wcag2a", "wcag2aa", "wcag21a", "wcag21aa"], // For available values, see https://www.deque.com/axe/core-documentation/api-documentation/#user-content-axe-core-tags
  "resultTypes": ["incomplete", "violations"],
  "filePath": "./urls.txt",
  "encoding": "utf8",
  "locale": "ja"
}
```
